### ISSUE TITLE
Tracking issues

### ISSUE THREAD

**Issue by JonathonBartle** [JonathonBartle/hello-world#2]
How do I reference issues when i do a fix?

**Comment by JonathonBartle** [comment]
Closed in error!

**Comment by JonathonBartle** [comment]
Mmm - no obvious way of linking issue with a new version of the code. Time to Google!...

**Comment by JonathonBartle** [comment]
For example, to close an issue numbered 123, you could use the phrase "Closes #123" or "Closes: #123" in your pull request description or commit message. Once the branch is merged into the default branch, the issue will close.

**Comment by JonathonBartle** [comment]
Reopen
